test/e2e/testnets/testnet.go

```diff
@@ -1,4 +1,4 @@
-package e2e
+package testnets
```

Comment:
[not blocking] I think the convention is for Go package names to be singular not plural

Ref:
- https://forum.golangbridge.org/t/naming-conventions/11180
- https://go.dev/blog/package-names